winml/lib/Api.Core/WinMLAdapter.cpp

```diff
@@ -50,7 +52,7 @@ class InferenceSessionProtectedLoadAccessor : public onnxruntime::InferenceSessi
     return onnxruntime::InferenceSession::Load(std::move(p_model_proto));
   }
   const onnxruntime::SessionState& GetSessionState() {
-    return session_state_;
+    return onnxruntime::InferenceSession::session_state_;
```

Comment:
why did you have to do this?

Reply:
Change wasn't intended, I did this accidentally when I was coding.